wsmaster/che-core-api-auth/src/main/java/org/eclipse/che/security/oauth1/OAuthAuthenticator.java

```diff
@@ -263,7 +263,7 @@ public String computeAuthorizationHeader(
       return computeAuthorizationHeader(
           requestMethod, requestUrl, credentials.token, credentials.tokenSecret);
     }
-    return null;
+    return "";
```

Comment:
Why do we need to change it?

Reply:
To avoid `null` when building a request since I removed throwing the `Unauthorised` exception if token is not found: https://github.com/eclipse-che/che-server/pull/428/files#diff-9069ac04b42d1a691656d5f2b427a441250195398173a8035c671b261c5a2276L406-L411

Reply:
returning null is a bad idea in general - https://stackoverflow.com/questions/1274792/is-returning-null-bad-design 
in general, I do believe it would be better to use `Optional` for this scenario